packages/common/src/rule-engine/operators/helpers.ts

```diff
@@ -1,11 +1,11 @@
-import { primitive, ConditionFn, BetweenParams, LastYearsParams, ExistsParams } from './types';
+import { ConditionFn, BetweenParams, LastYearsParams, ExistsParams, Primitive } from './types';
 import { TOperation } from '../types';
 import { ZodSchema } from 'zod';
 import { BetweenSchema, LastYearsSchema, PrimitiveSchema } from './schemas';
 import { ValidationFailedError } from '../errors';
-import { isEmpty } from 'lodash';
+import isEmpty from 'lodash.isempty';
```

Comment:
Why do we need this change?

Reply:
lodash was exists on workflow service, since its move to 'common' we dont have the full lodash library